doc/relevancy.md

```diff
@@ -0,0 +1,460 @@
+| `with_index_synonyms`    | ✖                  | New weighting query uses this for FIELD.synonym                         | ✔                | ✔            | standard                 | ✖    | ✔         |              | ✖                   | ✖                 | Index synonym list  | Synonyms                     | ✔                | ✔               | ✖             | ✖                 |
+| `with_search_synonyms`   | ✖                  | New weighting query uses this at search time                            | ✔                | ✔            | standard                 | ✖    | ✔         |              | ✖                   | ✖                 | Search synonym list | Synonyms                     | ✔                | ✔               | ✖             | ✖                 |
+| `exact_match`            | ✖                  | Used by best_bet_exact_match_text                                       | ✔                | ✖            | standard                 | ✔    | ✔         | ✖            | ✖                   | ✖                 | ✖                   | ✖                            | ✖                | ✖               | ✖             | ✖                 |
+| `best_bet_stemmed_match` | ✖                  | Used for triggering best bets                                           | ✔                | ✔            | standard                 | ✖    | ✔         |              | ✖                   | ✖                 | ✖                   | ✖                            | ✔                | ✔               | ✖             | ✖                 |
+| `spelling_analyzer`      | ✖                  | Used in spelling_text field                                             | ✔                | ✔            | standard                 | ✖    | ✔         |              | ✖                   | ✖                 | ✖                   | ✖                            | ✖                | ✖               | shingle       | ✖                 |
+| `string_for_sorting`     | ✖                  | Used for storing a sortable subfield                                    | ✔                | ✔            | keyword                  | ✔    | ✔         | ✖            | ✖                   | ✖                 | ✖                   | ✖                            | ✖                | ✖               | ✖             | ✖                 |
+
+
+The main analyzer used at the moment is called `with_search_synonyms`.
+
+```
+with_search_synonyms:
+  type: custom
+  tokenizer: standard
+  filter: [standard, asciifolding, lowercase, search_synonym, stop, stemmer_override, stemmer_english]
+  char_filter: [normalize_quotes, strip_quotes]
+```
+
+These are the steps, ignoring asciifolding, which have been added:
```

Comment:
I found this a bit confusing. Maybe it would help if you reference the various elements of the custom analyser under each of the smaller headings. Eg Under`Split into tokens` mention that this is defined by the `filter: [stop]` .